orchestrator/careplancontributor/service.go

```diff
@@ -161,8 +202,8 @@ func (s Service) readServiceRequest(localFHIR fhirclient.Client, serviceRequestR
 func (s Service) createCarePlan(patient fhir.Patient) (*fhir.CarePlan, error) {
 	carePlan := fhir.CarePlan{
 		Subject: fhir.Reference{
-			Type:       to.Ptr("Patient"),
-			Identifier: &patient.Identifier[0], // TODO: is this the right way/one?
+			Type:      to.Ptr("Patient"),
+			Reference: to.Ptr(*patient.Id),
```

Comment:
`Id` is already a pointer, no need to dereference and make a pointer again
```suggestion
			Reference: patient.Id,
```